.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,41 @@
+name: CI Tests
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
As a side note, maybe we should rename to top branch to `main`

Reply:
Would love to, I'll go do that.